OpenEdXMobile/src/main/java/org/edx/mobile/view/Router.java

```diff
@@ -361,4 +367,29 @@ public void showExploreSubjects(@NonNull Context context) {
     public void showWebViewActivity(@NonNull Activity activity, @NonNull String url, @Nullable String title) {
         activity.startActivity(WebViewActivity.newIntent(activity, url, title));
     }
+
+    public void showFeedbackScreen(FragmentActivity activity, String subject) {
+        String to = config.getFeedbackEmailAddress();
+
+        String osVersionText = String.format("%s %s", activity.getString(R.string.android_os_version),
+                android.os.Build.VERSION.RELEASE);
+        String appVersionText = String.format("%s %s", activity.getString(R.string.app_version),
+                BuildConfig.VERSION_NAME);
+        String deviceModelText = String.format("%s %s", activity.getString(R.string.android_device_model),
+                Build.MODEL);
+        String feedbackText = activity.getString(R.string.insert_feedback);
+        String body = osVersionText + "\n" + appVersionText + "\n" + deviceModelText + "\n\n" + feedbackText;
+        EmailUtil.openEmailClient(activity, to, subject, body, config);
+    }
+
+    public void openAppPageInPlayStore(@NonNull Context context) {
+        String packageName = context.getPackageName();
+        try {
+            context.startActivity(new Intent(Intent.ACTION_VIEW,
+                    Uri.parse("market://details?id=" + packageName)));
+        } catch (android.content.ActivityNotFoundException anfe) {
+            context.startActivity(new Intent(Intent.ACTION_VIEW,
+                    Uri.parse("https://play.google.com/store/apps/details?id=" + packageName)));
```

Comment:
This should be accessible in the config
https://github.com/edx/edx-app-android/blob/master/OpenEdXMobile/src/main/java/org/edx/mobile/util/Config.java#L448